src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const treeItems = document.querySelectorAll('.tree li');
+
+treeItems.forEach((item) => {
+  const textNode = item.firstChild;
+
+  const category = document.createElement('span');
+
+  category.textContent = textNode.textContent.trim();
+
+  item.replaceChild(category, textNode);
+
+  const changeCategoryListVisible = () => {
+    const categoryList = item.querySelector('ul');
+
+    if (categoryList) {
+      if (categoryList.classList.contains('hidden')) {
+        categoryList.classList.remove('hidden');
+      } else {
+        categoryList.classList.add('hidden');
+      }
```

Comment:
```suggestion
      categoryList.classList.toggle('hidden');
```
There is a `toggle` method, so it can be simplified
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList